integration/start-restart-nodeB.sh

```diff
@@ -4,11 +4,11 @@ mkdir -p ./logs
 echo Press Ctrl-C to exit
 for ((;;))
 do 
-	./bin/skywire-visor ./integration/intermediary-nodeB.json --tag NodeB 2>> ./logs/nodeB.log >> ./logs/nodeB.log &
-	echo node starting NodeB
+	./bin/skywire-visor ./integration/intermediary-visorB.json --tag VisorB 2>> ./logs/visorB.log >> ./logs/visorB.log &
+	echo visor starting VisorB
 	sleep 25
-	echo Killing NodeB on $(ps aux |grep "[N]odeB" |awk '{print $2}')
+	echo Killing VisorB on $(ps aux |grep "[N]odeB" |awk '{print $2}')
```

Comment:
`grep "[V]isorB"` (needs to be changed everywhere)

Reply:
done